tests/test_tables.py

```diff
@@ -1157,6 +1157,9 @@ def test_remove_column_type(table):
 # Create #
 ##########
 
+def test_empty_labels_none():
```

Comment:
I think you've actually stumbled on a bug here - this shouldn't have thrown an Error. Can you remove this test case and I'll create a ticket in the backlog to remove this method altogether.

Reply:
Update: I've solved this bug in #570